sycl/source/detail/event_impl.cpp

```diff
@@ -295,9 +292,19 @@ uint64_t
 event_impl::get_profiling_info<info::event_profiling::command_start>() {
   checkProfilingPreconditions();
   if (!MHostEvent) {
-    if (MEvent)
-      return get_event_profiling_info<info::event_profiling::command_start>(
-          this->getHandleRef(), this->getPlugin());
+    if (MEvent) {
+      auto startTime =
```

Comment:
Inconsistent variable naming in the single block of code

Reply:
Fixed in follow up patch